### ISSUE TITLE
[examples] example_imu_waist_down does not compile

### ISSUE THREAD

**Issue by liesrock** [robotology-playground/OpenSoT#45]
Hi,
this is just to report that while compiling the OpenSoT example I had the following error:

```
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp: In function ‘int main(int, char**)’:
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp:72:5: error: ‘fromKDLRotationToYARPMatrix’ is not a member of ‘cartesian_utils’
     cartesian_utils::fromKDLRotationToYARPMatrix(orientation, imu_orientation);
     ^
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp:73:21: error: ‘class iDynUtils’ has no member named ‘setIMUOrientation’
     robot.idynutils.setIMUOrientation(imu_orientation, robot.getIMU()->getReferenceFrame());
                     ^
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp:73:72: error: ‘class yarp_IMU_interface’ has no member named ‘getReferenceFrame’
     robot.idynutils.setIMUOrientation(imu_orientation, robot.getIMU()->getReferenceFrame());
                                                                        ^
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp:146:9: error: ‘fromKDLRotationToYARPMatrix’ is not a member of ‘cartesian_utils’
         cartesian_utils::fromKDLRotationToYARPMatrix(orientation, imu_orientation);
         ^
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp:149:25: error: ‘class iDynUtils’ has no member named ‘setIMUOrientation’
         robot.idynutils.setIMUOrientation(imu_orientation, robot.getIMU()->getReferenceFrame());
                         ^
/home/lucamuratore/robotology-superbuild/external/OpenSoT/examples/example_imu_waist_down.cpp:149:76: error: ‘class yarp_IMU_interface’ has no member named ‘getReferenceFrame’
         robot.idynutils.setIMUOrientation(imu_orientation, robot.getIMU()->getReferenceFrame());
                                                                            ^

```

It looks like it depends on some missing functions in cartisian_utils and idynutils.

**Comment by liesrock** [comment]
we are in the devel branch, commit 9f75009685

**Comment by liesrock** [comment]
This is solved checking out in commit e3c3b1b of idynutils library.

Thanks @EnricoMingo
